ReactiveCocoa/AppKit/NSTextField.swift

```diff
@@ -0,0 +1,34 @@
+import ReactiveSwift
+import AppKit
+import enum Result.NoError
+
+extension Reactive where Base: NSTextField {
+	/// A signal of values in `String` from the text field upon any changes.
+	public var continuousStringValues: Signal<String, NoError> {
+		var signal: Signal<String, NoError>!
```

Comment:
This pattern is used a lot. I created https://github.com/ReactiveCocoa/ReactiveSwift/pull/66 in `ReactiveSwift` so that at least we don't repeat it there, we might want to do something similar in `ReactiveCocoa`, but this is fine for now.